tests/cli/CleanCest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CleanCest
```

Comment:
```suggestion
<?php

declare(strict_types=1);

final class CleanCest
```

Reply:
I suggested including `final` as well since the other classes in this folder are declared like this. This was done because there is no implicit intention of inheritance here. It would be good to be congruent and decide whether to have final test classes or not. But the point is, they should be the same.

Reply:
Done